app/lib/dttp/code_sets/programme_subjects.rb

```diff
@@ -19,26 +19,26 @@ module ProgrammeSubjects
         "English as a second or other language" => { entity_id: "" },
         "English" => { entity_id: "c08274df-181e-e711-80c8-0050568902d3" },
         "French" => { entity_id: "cc8274df-181e-e711-80c8-0050568902d3" },
-        "Further education" => { entity_id: "" },
+        # "Further education" => { entity_id: "" },
         "Geography" => { entity_id: "9c8274df-181e-e711-80c8-0050568902d3" }, # duplicate
         "German" => { entity_id: "ce8274df-181e-e711-80c8-0050568902d3" },
         "Health and social care" => { entity_id: "9a8274df-181e-e711-80c8-0050568902d3" },
         "History" => { entity_id: "58a12838-b3cf-e911-a860-000d3ab1da01" }, # duplicate
         "Italian" => { entity_id: "d08274df-181e-e711-80c8-0050568902d3" }, # verify
-        "Japanese" => { entity_id: "" },
-        "Mandarin" => { entity_id: "" },
+        "Japanese" => { entity_id: "6ca12838-b3cf-e911-a860-000d3ab1da01" }, # verify
```

Comment:
These are the same. Is that right?

Reply:
For now. There isn't a DTTP equivalent, so after speaking to Ben, we decided to map it to modern languages. We might change this again after our DTTP meeting tomorrow.